### PR TITLE
Bug 1522677 – Card Grid refinements

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
If this looks good feel free to land it while I'm out.

File: content-src/components/DiscoveryStreamComponents/CardGrid/_CardGrid.scss [hunk 1]
```diff
@@ -1,11 +1,23 @@
+$col4-header-line-height: 20;
+$col4-header-font-size: 13;
```
Comment (ScottDowne) [inline content-src/components/DiscoveryStreamComponents/CardGrid/_CardGrid.scss]: I don't see this actually being set to the font size, just added to the limit-visibile-lines mixin.

**Comment by ScottDowne** [comment]
One nit, and I'm also not sure what's going on with "Vertical gap between rows should be 24px" how do I test that, or what should I be looking for?

**Comment by ScottDowne** [comment]
Given Gavin is out for the day, I think merging this and doing a followup for the font size that might be broken when Gavin's back is fine.

**Comment by Mardak** [comment]
The hover state of borderless cards should have no border. Also seems like dev-test-all 2/3 column "Card" vs "Borderless Card" both are actually set as border: no-border !

File: content-src/components/DiscoveryStreamComponents/CardGrid/_CardGrid.scss (hunk 1 above)
Comment (gvn) [reply content-src/components/DiscoveryStreamComponents/CardGrid/_CardGrid.scss]: That mixin now takes the font size for some internal calculations and emits it as well.

**Pull request merged**